client/my-sites/woocommerce/woocommerce-colophon.tsx

```diff
@@ -1,24 +1,29 @@
 import { recordTracksEvent } from '@automattic/calypso-analytics';
 import { useTranslate } from 'i18n-calypso';
-import ExternalLink from 'calypso/components/external-link';
 import WooCommerceLogo from './woocommerce-logo';
 
-function WooCommerceColophon() {
+export interface WooCommerceColophonProps {
+	wpcomDomain: string;
+}
+
+function WooCommerceColophon( props: WooCommerceColophonProps ) {
 	const translate = useTranslate();
+	const { wpcomDomain } = props;
+	const woocommercePluginURL = `http://calypso.localhost:3000/plugins/woocommerce/${ wpcomDomain }`;
 
 	const onClick = () => {
 		recordTracksEvent( 'calypso_woocommerce_woocommercecolophon_click' );
 	};
 
 	return (
 		<div className="woocommerce-colophon">
-			<ExternalLink icon={ false } onClick={ onClick } href="https://woocommerce.com">
+			<a onClick={ onClick } href={ woocommercePluginURL }>
```

Comment:
Probably best to keep External link here was there a reason to change?

Reply:
Since this isn't an external URL thought this wasn't a good use case for `ExternalLink`

Reply:
Ah yes, you're right!